mlir/include/mlir/Analysis/AffineStructures.h

```diff
@@ -139,6 +139,16 @@ class FlatAffineConstraints {
   // returns true, no integer solution to the equality constraints can exist.
   bool isEmptyByGCDTest() const;
 
+  // \returns True if the set of constraints has no integer solution, and False
+  // if it does have a solution.
+  bool isIntegerEmpty() const;
+
+  // Find a sample point satisfying the constraints. This uses a branch and
```

Comment:
why not /// instead of // ?

Reply:
For consistency with the rest of the file.

Reply:
ok.